community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -102,6 +102,10 @@
             "allows an upgrade to be performed.")
     public static final Setting<Boolean> allow_store_upgrade = setting("dbms.allow_format_migration", BOOLEAN, FALSE );
 
+
+    @Description( "Specifies custom storage format name. If not set default format will be used." )
+    public static final Setting<String> record_format = setting( "dbms.record_format", Settings.STRING, "" );
```

Comment:
I would expect this to be an enum setting (like `ha.branched_data_policy`). Currently the user has no way to know what valid values are.